webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtVmware.js

```diff
@@ -48,6 +48,41 @@ const VirtVmware = ({ vm }) => (
         {vm.corespersocket}
       </DescriptionListDescription>
     </DescriptionListGroup>
+    {Object.values(vm.disks).map((disk, index) => (
+      <DescriptionListGroup key={`disk-${index}`}>
+        <DescriptionListTerm>{__('Disk capacity')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {NumberToHumanSize(disk.capacity * 1024 ** 2, {
```

Comment:
Not sure if this is the right calculation as it shows up as  `NaN undefined` when I have `disk.capacity=3221225472`. It calculates correctly in the legacy host view

Reply:
you are right. Fixed it. thank you.